Shell/Options.hpp

```diff
@@ -490,10 +490,9 @@ class Options
   };
 
   enum class QuestionAnsweringMode : unsigned int {
-    ANSWER_LITERAL = 0,
-    FROM_PROOF = 1,
-    SYNTHESIS = 2,
-    OFF = 3
+    PLAIN = 0,
```

Comment:
I think `FROM_PROOF` is no more, right? Did the unused `FROM_PROOF` code get removed already?

Reply:
I think there was no code. Just plans ...